exspy/tests/models/test_eelsmodel.py

```diff
@@ -73,19 +73,44 @@ def test_gos_hydrogenic(self):
 
     def test_gos_gosh(self):
         m = self.s.create_model(auto_add_edges=True, GOS="gosh")
```

Comment:
To test that it does warn as expected and capture the warning to avoid adding more warning when running the test suite.
```suggestion
        with pytest.warns(VisibleDeprecationWarning):
            m = self.s.create_model(auto_add_edges=True, GOS="gosh")
```